Nebula.Web/src/app/pages/field-definition-edit/field-definition-edit.component.html

```diff
@@ -36,15 +36,9 @@ <h2 class="d-inline-block">
         <div class="col-12">
             <editor
                 #tinyMceEditor
-                [init]="{
-                    toolbar: false,
-                    menubar: false,
-                    plugins: 'quickbars',
-                    promotion: false
-                }"
+                [init]="tinyMceConfig"
```

Comment:
Hey Liz! This is a good opportunity to add the `tinyMceConfig` pipe to this project. Even though we're using the `ngAfterViewInit()` in the component, there's still no guarantee that the `ViewChild` for the editor is actually hydrated when we set up the config. Here is a link back to that pipe for KAP: https://github.com/sitkatech/ManitobaEFP/blob/5fcdf87c4a0197bdc62d0b999dc5b956dcaf49e6/Source/ManitobaEFP.Web/src/app/shared/pipes/tiny-mce-config.pipe.ts